components/resources/wallet_strings.grdp

```diff
@@ -924,10 +924,11 @@
   <message name="IDS_BRAVE_SWAP_HALF" desc="Amount preset button text for half">Half</message>
   <message name="IDS_BRAVE_SWAP_MAX" desc="Amount preset button text for max">Max</message>
   <message name="IDS_BRAVE_SWAP_ENTER_AN_AMOUNT" desc="Text prompting to enter an amount">Enter an amount</message>
-  <message name="IDS_BRAVE_SWAP_FINDING_PRICE" desc="Loader text while quotes are fetched">Finding the best quote...</message>
+  <message name="IDS_BRAVE_WALLET_FETCHING_QUOTE" desc="Loader text while quotes are fetched">Fetching quote</message>
   <message name="IDS_BRAVE_WALLET_RECEIVE_ESTIMATE" desc="Swap receive estimate label">Receive (est.)</message>
   <message name="IDS_BRAVE_WALLET_FROM_TOKEN" desc="Swap from token label">From token</message>
   <message name="IDS_BRAVE_WALLET_TO_TOKEN" desc="Swap to token label">To token</message>
+  <message name="IDS_BRAVE_WALLET_NEW_QUOTE_IN" desc="Swap new quote in label">New quote in <ph name="COUNTDOWN_BEFORE">$1</ph>$countdown<ph name="COUNTDOWN_AFTER">$2</ph></message>
```

Comment:
Seems like countdown is also a placeholder? Would be easier for the localizers to understand if it were: `<ph name="COUNTDOWN_BEFORE">$1</ph><ph name="COUNTDOWN">$1<ex>00:05</ex></ph><ph name="COUNTDOWN_AFTER">$2</ph>`. If this makes things too complicated on React side, then at least let's add something to the `desc` that explains that `$countdown` would be replaced with a digital counting down clock or such.

Reply:
@mkarolin good point, I have updated this with a the `<ex></ex>` tags! https://github.com/brave/brave-core/compare/47906f3f4a9e06a43ecd6b666fa27302ebe50b39..f7de84c2d9b6178d713dd1e7e3f63d323cb513ed